styles.css

```diff
@@ -655,3 +655,11 @@ footer .ending-line {
     display: none;
   }
 }
+
+@media screen and (min-width: 1100px) {
+  .works .works-section {
+    display: grid;
+    grid-template-columns: 400px 400px 400px;
+    margin-left: 110px;
```

Comment:
- Hey, great job on adjusting the issue as pointed out by the previous reviewer, however, you have too much space inside the container holding each card. The cards are not taking the full width of the container. Also, you've used too many margin-left which is actually not making your cards perfectly in the center of the container.
   - What you can do to improve this is to try removing all those margin-left and try to adjust your styles so these cards can be centered correctly. 
   - Try using a fluid measurement like this  `grid-template-columns: repeat(3, 1fr);` so the cards can take up the whole width. Please, check your design closely to fix additional alignment issues it will have when adjusting the styles. see the images below:


*Your design with too much space*
![Screenshot from 2023-02-23 16-52-14](https://user-images.githubusercontent.com/59397159/220964665-75ec70b0-64c7-4367-9a0b-f3c466d0a75c.png)